.github/workflows/container_tests.yml

```diff
@@ -0,0 +1,92 @@
+          echo $MOD_INIT > mod_init
+          echo $PATH > path
+          if [ ! -z $MODULESHOME ]; then echo $MODULESHOME > moduleshome; fi
+
+    # see https://github.com/apptainer/singularity/issues/5390#issuecomment-899111181
+    - name: install Singularity
+      run: |
+        # install alien, which can be used to convert RPMs to Debian packages
+        sudo apt-get install alien
+        alien --version
+        # determine latest version of Singularity available in EPEL, and download RPM
+        singularity_rpm=$(curl -sL https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/s/ | grep singularity | sed 's/.*singularity/singularity/g' | sed 's/rpm.*/rpm/g')
+        curl -OL https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/Packages/s/${singularity_rpm}
+        # convert Singularity RPM to Debian package, and install it
+        sudo alien -d ${singularity_rpm}
+        sudo apt install ./singularity*.deb
+        singularity --version
+
+    - name: install sources
+      run: |
+          # install from source distribution tarball, to test release as published on PyPI
+          python setup.py sdist
+          ls dist
+          export PREFIX=/tmp/$USER/$GITHUB_SHA
+          pip install --prefix $PREFIX dist/easybuild-framework*tar.gz
+          pip install --prefix $PREFIX https://github.com/easybuilders/easybuild-easyblocks/archive/develop.tar.gz
+
+    - name: run test
+      run: |
+          # run tests *outside* of checked out easybuild-framework directory,
+          # to ensure we're testing installed version (see previous step)
+          cd $HOME
+          # initialize environment for modules tool
+          if [ -f $HOME/moduleshome ]; then export MODULESHOME=$(cat $HOME/moduleshome); fi
+          source $(cat $HOME/mod_init); type module
+          # make sure 'eb' is available via $PATH, and that $PYTHONPATH is set (some tests expect that);
+          # also pick up changes to $PATH set by sourcing $MOD_INIT
+          export PREFIX=/tmp/$USER/$GITHUB_SHA
+          export PATH=$PREFIX/bin:$(cat $HOME/path)
+          export PYTHONPATH=$PREFIX/lib/python${{matrix.python}}/site-packages:$PYTHONPATH
+          eb --version
+          # create $HOME/.rpmmacros, see also https://github.com/apptainer/singularity/issues/241
+          echo '%_var /var' > $HOME/.rpmmacros
+          echo '%_dbpath %{_var}/lib/rpm' >> $HOME/.rpmmacros
+          # build CentOS 7 container image for bzip2 1.0.8 using EasyBuild;
+          # see https://docs.easybuild.io/en/latest/Containers.html
+          curl -OL https://raw.githubusercontent.com/easybuilders/easybuild-easyconfigs/develop/easybuild/easyconfigs/b/bzip2/bzip2-1.0.8.eb
+          export EASYBUILD_CONTAINERPATH=$PWD
+          export EASYBUILD_CONTAINER_CONFIG='bootstrap=yum,osversion=7'
+          eb bzip2-1.0.8.eb --containerize --experimental --container-build-image
```

Comment:
`bzip2` has a couple of deps including `binutils`, which  is why this CI is taking 10 mins. Are you concerned about ensuring `--robot` is working? If not, maybe `zlib` is enough...

Reply:
I'm not too concerned about that, since this is a separate workflow, and the framework test suite also takes ~10min.

If it's really a nuisance, we can reconsider later?